.github/workflows/test.yml

```diff
@@ -10,6 +10,23 @@ on:
     - cron:  '0 * * * *'
 
 jobs:
+  build_and_test_libstatistics_collector:
```

Comment:
What about `metrics_statistics_msgs`? Since that will also be a ros2 core dependency, do we need to add the same testing for it also?

Reply:
Good point, I'll add it here (or we have a separate job)

Reply:
Separate job would be better imo.